A3A/addons/core/functions/init/fn_initClient.sqf

```
@@ -245,13 +245,14 @@ player addEventHandler ["GetInMan", {
 	_unit = _this select 0;
 	_veh = _this select 2;
 	_exit = false;
-	if (isMultiplayer) then {
 		if !([player] call A3A_fnc_isMember) then {
 			_owner = _veh getVariable "ownerX";
-			if (!isNil "_owner") then {
-				if (_owner isEqualType "") then {
-					if ({getPlayerUID _x == _owner} count (units group player) == 0) then {
-						["Warning", "You cannot board other player vehicle if you are not in the same group."] call A3A_fnc_customHint;
+			_locked = _veh getVariable "locked";
+			if (_locked isequalto true) then {
```

Similarly this is an actual nil comparison, which tends to silently fail but isn't good practice. Better to use nil/true + default false on getVariable.

---

sqf is not statically typed, we cant be sure that a outside variable will be of a specific type so better to have isEqualTo which is a type safe comparison

---

If there's a risk that it's a variable from outside (which there is, I guess), we should be renaming it to A3A_locked.